src/lib/InstantSearch.js

```diff
@@ -90,32 +90,38 @@ Usage: instantsearch({
     this.widgets.push(widget);
   }
 
-  removeWidget(widget) {
-    if (
-      !this.widgets.includes(widget) ||
-      typeof widget.dispose !== 'function'
-    ) {
-      throw new Error(
-        'The widget you tried to remove does not implement the dispose method, therefore it is not possible to remove this widget'
-      );
+  removeWidget(widgets) {
```

Comment:
Should rename it `removeWidgets()` @bobylito WDYT?

Reply:
In my experience I like to have 2 API's and not have polymorhpic methods. In this case, I like to have `removeWidget` which accepts a single widget and `removeWidgets` that accepts an array of widget. The implementation of `removeWidget` is trivial because it can use `removeWidgets`. WDYT?

Reply:
Good 👍